### PR TITLE
mocktikv: fix mocktikv kv scan bug

### PR DESCRIPTION
**Pull request by crazycs520**
### What problem does this PR solve? 
Fix mocktikv  `handleKvScan` bug. 

### Check List 

Tests 

Code changes

 - Has exported function/method change

Side effects

Related changes

### CODE REVIEW

File: store/mockstore/mocktikv/rpc.go [hunk 1]
```diff
@@ -248,8 +248,8 @@ func (h *rpcHandler) handleKvScan(req *kvrpcpb.ScanRequest) *kvrpcpb.ScanRespons
 	if !h.checkKeyInRegion(req.GetStartKey()) {
 		panic("KvScan: startKey not in region")
 	}
-	endKey := h.endKey
-	if len(req.EndKey) > 0 && (len(endKey) == 0 || bytes.Compare(req.EndKey, endKey) < 0) {
+	endKey := MvccKey(h.endKey).Raw()
```
Comment (lonng) [inline store/mockstore/mocktikv/rpc.go]:
```suggestion
	endKey := h.rawEndKey
```

Comment (crazycs520) [reply store/mockstore/mocktikv/rpc.go]: @lonng , `h.rawEndKey` maybe doesn't set. And If alway set `h.rawEndKey`when set `h.endKey`, you will got some other error when `make test`.

**Comment by lonng** [comment]
LGTM

**Pull request merged**